modules/core/include/opencv2/core/utils/logger.configuration.hpp

```diff
@@ -0,0 +1,28 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_LOGGER_CONFIGURATION_HPP
+#define OPENCV_LOGGER_CONFIGURATION_HPP
+
+//! @addtogroup core_logging
+//! @{
+
+namespace cv {
+namespace utils {
+namespace logging {
```

Comment:
Probably namespace statements can be omitted here.

Reply:
Removed